clang/include/clang/Basic/Attr.td

```diff
@@ -2355,6 +2355,13 @@ def NoInline : DeclOrStmtAttr {
   let SimpleHandler = 1;
 }
 
+def NoOutline : DeclOrStmtAttr {
+  let Spellings = [Clang<"nooutline">];
```

Comment:
```suggestion
  let Spellings = [Clang<"no_outline">];
```

perhaps?  Makes it... somewhat consistent with the `force_inline`/etc type attributes, and makes it a better mental 'parse'.

Reply:
I wrote a longer comment about this, see above.